### PR TITLE
[CI] Add tests GitHub Action workflow

### PR DESCRIPTION
**Pull request by TomasVotruba**
- composer - add PHP 7.3 required version, based on syntax
- add tests on PHP 7.3 to 8.1

### CODE REVIEW

**Comment by TomasVotruba** [comment]
This will CI with tests to run on every PR.
It will be invoked after merge, so if something is failing, I'll send follow up PR :)

**Comment by TomasVotruba** [comment]
Ready for review & merge :heavy_check_mark:

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,10 @@
             "helpers/json.php"
         ]
     },
-    "require": {},
+    "require": {
+        "php": ">=7.3"
+    },
     "require-dev": {
-        "phpunit/phpunit": "^7"
+        "phpunit/phpunit": "^9.5"
```
Comment (TomasVotruba) [inline composer.json:25]: The PHPUnit 9.5 runs on PHP 7.3+, so we can use it right now

https://phpunit.de/supported-versions.html

**Comment by donjajo** [comment]
Thank you very much, @TomasVotruba for this contribution :)

**Comment by TomasVotruba** [comment]
Hi @donjajo ,

I'm waiting for these features to be tagged so we download only stable packages with composer :)
Do you think you could tag this one?

Thanks

**Comment by donjajo** [comment]
> Hi @donjajo ,
> 
> I'm waiting for these features to be tagged so we download only stable packages with composer :) Do you think you could tag this one?
> 
> Thanks

Done, thanks

**Comment by TomasVotruba** [comment]
Thank you :cake:

**Pull request merged**